packages/design-preview/src/components/style.scss

```diff
@@ -24,6 +24,7 @@ $break-design-preview: 1024px;
 
 .design-preview__sidebar {
 	align-items: center;
+	background-color: #fdfdfd;
```

Comment:
Is there a colour variable you could use here? That way CSS is more maintainable and Calypso colour themes would continue working as expected.

Reply:
I considered this before but I couldn't find a suitable colour variable for it 😅 So, I define a new one called `--color-body-background` for the stepper framework, what do you think?

Reply:
I ran into this issue as well. @simison AFAIK `#fdfdfd` is not a Calypso color, but still used as background color in [many places](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso+%23fdfdfd&type=code) cc: @SaxonF

Reply:
I'd still include it in [calypso-color-schemes](https://github.com/Automattic/wp-calypso/tree/trunk/packages/calypso-color-schemes) package, especially if it's used in multiple places. Otherwise you'll have seemingly _random colour_ (as we just established this is random colour outside defined palette) in seemingly _random files_. Canonical place seems still better to me. :-)

Reply:
Aside, there could be something close enough in the palette that this could be switched over?

Don't let this hold from merging btw! This is perfectly fine for a follow-up and not urgent.